general/toc.yml

```diff
@@ -2,7 +2,7 @@
   items:
   - uid: clients-css-customization
   - uid: clients-codec-support
-  - uid: clients-installing-kodi
+  - uid: clients-kodi
   - uid: clients-installing-mopidy
```

Comment:
You might want to rename this one as well in the future.

Reply:
Yeah, definitely. I could also add a page for the mpv shim as well.